debian/jitsi-meet-prosody.postinst

```diff
@@ -155,10 +155,17 @@ case "$1" in
         MAIN_MUC_PATTERN="Component \"conference.$JVB_HOSTNAME\" \"muc\""
         if ! grep -A 2 -- "${MAIN_MUC_PATTERN}" $PROSODY_HOST_CONFIG | grep -q "restrict_room_creation" ;then
             sed -i "s/${MAIN_MUC_PATTERN}/${MAIN_MUC_PATTERN}\n    restrict_room_creation = true/g" $PROSODY_HOST_CONFIG
+            PROSODY_CONFIG_PRESENT="false"
```

Comment:
should this be here? it looks like this will force a restart later in the script if restrict_room_creation wasn't defined

Reply:
Yep, that was a miss in my previous PR